dotCMS/src/enterprise/java/com/dotcms/enterprise/publishing/remote/handler/ContentHandler.java

```diff
@@ -331,14 +333,29 @@ private void handleContents(final Collection<File> contents, final File folderOu
 						// This operation (PUBLISH) publishes/un-publishes a Content that is NOT archived. The un-publishing
 						// is taken care of at the end of this handleContents() method
 						publish(content, folderOut, systemUser, wrapper, isHost,remoteLocalLanguages);
+						Logger.debug(this, "Content saved: " + content.getIdentifier()
```

Comment:
you should use Logger.debug(class, Supplier) such sa

Logger.debug(this, ()-> "Content saved: " + ...)

This makes the lazy creation of the msg so you safe mem if the debug is not activate